src/payment/strategies/ppsdk/step-handler/step-handler.ts

```diff
@@ -0,0 +1,5 @@
+import { PaymentsAPIResponse } from '../ppsdk-payments-api-response';
```

Comment:
Dummy code which will be expanded in on a future ticket: https://jira.bigcommerce.com/browse/PAYMENTS-6805

This hander will then discern response types and act accordingly.